dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -257,10 +257,14 @@ using DCPS::TimeDuration;
 using DCPS::SystemTimePoint;
 using DCPS::Serializer;
 using DCPS::Encoding;
-using DCPS::ENDIAN_BIG;
-using DCPS::ENDIAN_LITTLE;
 
-const Encoding sedp_encoding(Encoding::KIND_XCDR1, ENDIAN_LITTLE);
+namespace {
+  const Encoding sedp_encoding(Encoding::KIND_XCDR1, DCPS::ENDIAN_LITTLE);
+#ifdef OPENDDS_SECURITY
+  const Encoding secbinprop_encoding(Encoding::KIND_XCDR1, DCPS::ENDIAN_BIG);
```

Comment:
It's unclear from the name what this is

Reply:
Yeah, I thought `security_binary_property_encoding` was thought was a little long, but looking at it now I guess it's not too bad.

Reply:
Actually noting is using this... I guess I'll remove it?